packages/paste-core/components/toast/src/types.ts

```diff
@@ -29,17 +29,25 @@ export const ToastPropTypes = {
 export interface ToastPortalProps extends React.HTMLAttributes<HTMLDivElement> {
   children: NonNullable<React.ReactNode>;
   ref?: any;
+  // FIXME: Overrides token zIndex to fix bug with Console product.
+  __console_patch?: boolean;
 }
 export const ToastPortalPropTypes = {
   children: PropTypes.node.isRequired,
+  // FIXME: Overrides token zIndex to fix bug with Console product.
+  __console_patch: PropTypes.bool,
```

Comment:
Do you need it on `ToastPortal`?

Reply:
We do because the Toaster and ToastContainer ultimately pass this prop down to ToastPortal, where the ZIndex is applied.
![image](https://user-images.githubusercontent.com/1592327/111333124-c113ce00-8648-11eb-87c7-3e19fb144eab.png)